code/game/machinery/Beacon.dm

```diff
@@ -44,5 +41,3 @@
 		Beacon.invisibility = INVISIBILITY_MAXIMUM
```

Comment:
Shouldn't you still call `update_icon()` here where you're changing the invisibility?

Reply:
I don't think we need to `updateicon()` when `Beacon.invisibility`, which isn't `src.invisibility`, is updated

Reply:
Ah, didn't see that.